mitol-django-mail/testapp/README.md

```diff
@@ -0,0 +1,28 @@
+`mitol-django-mail` test app
+---
+
+
+#### Usage
+
+In the `mitol-django-mail directory`, you can test this app out by running:
+
+- `poetry run python manage.py migrate` - migrates sqlite db (gitignored)
+- `poetry run python manage.py shell`:
+
```

Comment:
I would still prefer some kind of note here indicating that you will only be able to send an email from the shell if you add settings for `MAILGUN_API_KEY` and `MAILGUN_SENDER_DOMAIN`. If you do add that, you might add a note that those values can be borrowed from any one of our Heroku apps that uses Mailgun

Reply:
Those values being invalid was intentional, it prevents an accidental send, and would cause mailgun to fail if somehow a test got through to make an actual call here.

I think I get what you're getting at - it's impossible to make this code work in a shell without taking on the risk of introducing secrets into `settings.py`, which could later be accidentally committed. I'm thinking maybe a gitignored `settings.dev.py` that's imported if it exists would provide a good avenue for overriding this stuff without having to modify a functional source file. It'll give us the same benefits of `.env` without having to lift the whole app into docker.